bazel/foreign_cc/luajit.patch

```diff
@@ -59,7 +59,7 @@ EOF
  
  @setlocal
 -@set LJCOMPILE=cl /nologo /c /O2 /W3 /D_CRT_SECURE_NO_DEPRECATE /D_CRT_STDIO_INLINE=__declspec(dllexport)__inline
-+@set LJCOMPILE=cl /nologo /c /W3 /D_CRT_SECURE_NO_DEPRECATE /D_CRT_STDIO_INLINE=__declspec(dllexport)__inline
++@set LJCOMPILE=cl /nologo /c /W3 /D_CRT_SECURE_NO_DEPRECATE /D_CRT_STDIO_INLINE=__declspec(dllexport)__inline /DLUAJIT_ENABLE_LUA52COMPAT
```

Comment:
Sorry, just curious, why we remove `/O2` here?

Reply:
`/O2` placed there was being applied to to debug and release build modes, it was removed here to enable us to target it to just release mode and disable optimizations entirely in debug

cc @wrowe

Reply:
You'll find /O2 is still applied appropriately later in the script, @dio

Reply:
Got it. Thanks, @wrowe.